bisection.py

```diff
@@ -216,7 +216,7 @@ def _build_lazy_tensor(self, commit: Commit, build_env: Dict[str, str]):
             lazy_tensor_path = os.path.join(self.srcpath, "lazy_tensor_core")
             command = "./scripts/apply_patches.sh"
             subprocess.check_call(command, cwd=lazy_tensor_path, env=build_env, shell=True)
-            command = "python setup.py install"
+            command = "DEBUG=1 USE_CUDA=1 USE_XNNPACK=0 python setup.py install"
```

Comment:
interesting.. what is the deal with XNNPACK?  I don't remember setting it to zero locally, but I also didn't notice any issues from it.

Reply:
I was told by @Krovatkin that this is needed...

Reply:
Just saw that the default configuration will turn it off. So I removed then. See: setup_build_env in the same file.